Threat-and-trust-model.md

```diff
@@ -1,35 +1,26 @@
 
-Unfortunately, CCv0 doesn't have a clear definition for the threat model of confidential
-container. If the confidential container is going to develop in a healthy manner, it is necessary
-to define and clarify the threat model as soon as possible in order to provide guidance and
-constraint on the components and architecture in the security principle of confidential container.
-Otherwise, it is difficult to follow an unified security principle among the components composing
-the confidential container. In addition, it's easy to fall into the constant discussions and
-clarifications.
+The confidential container open source project[1] has reached a milestone of CCv0, with great contribution from the community. However, there is no clear definition of threat model for confidential containers. It becomes clear that an agreed trust model and threat modeling definition is needed as a guidance to drive the next level of architecture defintion and development. Security objective and non-objective for confidential containers should also be clear to provide the security principles for the community to address any opens or questions in architecture and component level discussion. 
+
+In the next few sections, we will start the discussion of potentially different forms of confidential container based on different hardware technologies and software stacks, and identify the one we will address in this document; then discuss the trust model and threat model for this specific confiential container.
 
 ---
 
-## The forms of confidential container
+## Types of confidential container
 
-There is a number of ways to accomplish a confidential container, such as VM-based "vm-cc",
-MicroVM-based "microvm-cc", and Intel SGX Enclave based "enclave-cc".
+The objective of confidential containers is to protect containers. There are number of ways to accomplish that, based on different technologies. They can be VM based which uses VM for isoation, or Encalve based (e.g, Intel SGX). We category them into the three following models:
 
-"vm-cc" uses a virtual machine with hardware TEE capability to run a K8s node. "microvm-cc" uses
-a MicroVM with hardware TEE capability to run a Pod. "enclave-cc" uses a SGX Enclave to run a
-LibOS.
+- VM-based ("vm-cc"). Protection is at K8S node level. Each worker node is a virtual machine running inside a trusted exuection environment (TEE) with support from hardware isolation technology (Intel SGX, AMD-SEV).
+- MicroVM-based ("microvm-cc"). Protection is at Pod level. Each pod is running inside a MicroVM running inside TEE.
+- Enclave based ("enclave-cc"). Protection is at pod level. Each pod runs inside an enclave (e.g., Intel SGX).
```

Comment:
SGX-based CC runs application beyond LibOS, and LibOS runs in an enclave, and enclave is part of the hosting process, and the process is in container. So the protection level of Enclave based ("enclave-cc") would be "Process" or "Container".